aips/agg_is_at_least.md

```diff
@@ -0,0 +1,74 @@
+---
+aip: X
+title: Introduce aggregator_v2::is_at_least native API
```

Comment:
"Introduce native API to evaluate aggregator value is greater or equal to given amount" instead?

Reply:
updated